### DISCUSSION TITLE
Remember window location on Windows

### DISCUSSION THREAD

**Discussion by BooBerry** [ente-io/ente#2181] [Enhancements]
Currently the latest version of the Auth app always launches on the left side of the screen on Windows. Ideally I'd like the app to remember the last location of the window and open back to the same place with each app open. If you move the window and close the app and reopen it, it'll default back to the left position on the screen.

I'm using Windows 11 24H2 and a 1440p 165Hz monitor if that makes any difference.

**Comment by knItalian** [comment]
Yep，especially for users with multiple monitors or customized workspace layouts.
